VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1755,33 +1764,17 @@ public void callPlayerSeeked(long lastPostion, long newPosition, boolean isRewin
     }
 
     /**
-     * Displays controller is in PORTRAIT MODE, otherwise hides controller.
+     * Displays controller based on a source
```

Comment:
or are you of the opinion that, we just remove "based on a source" from the description only ?